lib/test_json_sink.c

```diff
@@ -135,7 +135,8 @@ tlog_test_json_sink_run(
 }
 
 bool
-tlog_test_json_sink(const char *name, const struct tlog_test_json_sink test)
+tlog_test_json_sink(const char *file, int line, const char *name,
+		                const struct tlog_test_json_sink test)
```

Comment:
Please align this line to the inside of the opening parentheses. I.e. like this:

    tlog_test_json_sink(const char *file, int line, const char *name,
                        const struct tlog_test_json_sink test)